tests/e2e/test_schema_e2e.py

```diff
@@ -0,0 +1,104 @@
+#  Copyright (c) "Neo4j"
+#  Neo4j Sweden AB [https://neo4j.com]
+#  #
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#  #
+#      https://www.apache.org/licenses/LICENSE-2.0
+#  #
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+from neo4j_genai.schema import (
+    query_database,
+    NODE_PROPERTIES_QUERY,
+    BASE_ENTITY_LABEL,
+    EXCLUDED_LABELS,
+    EXCLUDED_RELS,
+    REL_PROPERTIES_QUERY,
+    REL_QUERY,
+)
+
+
+def test_cypher_returns_correct_schema(driver):
```

Comment:
I think we can split these into 3 separate unit tests so it's more granular. Wdyt?

Reply:
Sure, we could do that. And also move the setup cypher to some kind of decorator.

Reply:
You can add and use a fixture like the ones in `conftest.py`

Reply:
[acbbe02](https://github.com/neo4j/neo4j-genai-python/pull/36/commits/acbbe0208b52bae330738c97b0934f9f01b1279e)